src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseNotAgreedRepaymentRespondentNotificationHandler.java

```diff
@@ -65,13 +71,20 @@ public List<CaseEvent> handledEvents() {
 
     @Override
     public Map<String, String> addProperties(final CaseData caseData) {
-        return new HashMap<>(Map.of(
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            CLAIM_LEGAL_ORG_NAME_SPEC, organisationDetailsService.getApplicantLegalOrganizationName(caseData)
-        ));
+        return (caseData.isApplicant1NotRepresented() && featureToggleService.isLipVLipEnabled())
+            ? new HashMap<>(Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                CLAIMANT_NAME, getPartyNameBasedOnType(caseData.getApplicant1())
+            ))
+            : new HashMap<>(Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                CLAIM_LEGAL_ORG_NAME_SPEC, organisationDetailsService.getApplicantLegalOrganizationName(caseData)
+            ));
     }
 
     private String addEmail(CaseData caseData) {
```

Comment:
I know that this method was implemented before but it shouldnt be named _addEmail_
It should be **_getEmail_**. It's not adding anything anywhere, its returning the email

Reply:
Changed the name as suggested above